config/initializers/bgs.rb

```diff
@@ -6,7 +6,7 @@
   # OpenSSL::SSL::VERIFY_PEER = OpenSSL::SSL::VERIFY_NONE
   BGS.configure do |config|
     config.application = Settings.bgs.application
-    config.client_ip = Socket.ip_address_list.detect(&:ipv4_private?).ip_address
+    config.client_ip = Socket.ip_address_list.detect(&:ipv4_private?)
```

Comment:
Why is this changed?

Reply:
@yoomlam good catch, my network settings are funky. i didn't meant to commit. ty.